zio-http/src/main/scala/zio/http/netty/ChannelFactories.scala

```diff
@@ -27,6 +27,7 @@ object ChannelFactories {
     def uring(implicit trace: Trace)  = serverChannel(new IOUringServerSocketChannel())
     def kqueue(implicit trace: Trace) = serverChannel(new KQueueServerSocketChannel())
 
+    implicit val trace: Trace = Trace.empty
```

Comment:
@adamgfraser what will be the recommended approach to create/pass a trace at this level ?

Reply:
@devsprint I think using an empty trace here is fine.